client/app/bundles/course/group/pages/GroupShow/index.jsx

```diff
@@ -93,7 +101,15 @@ const Category = ({
             }
           />
           {groups.map((group) => (
-            <GroupTableCard key={group.id} group={group} />
+            <GroupTableCard
+              key={group.id}
+              canManageCategory={canManageCategory}
+              group={group}
+              onManageGroups={() => {
+                dispatch({type: actionTypes.MANAGE_GROUPS_START})
```

Comment:
You can put this together inside handleGroupSelect so that the dispatches are grouped together. 

Also, any reason it's called `onManageGroups`? I understand that this function is to go edit a single group, so perhaps `onManageGroup`?

Finally, you should scroll to the top of <GroupUserManager /> component when the manage group button is clicked. Try having many groups and you click the last manage group. After clicking and redirecting to the page, you are now at the bottom at the page.

Reply:
`Finally, you should scroll to the top of component when the manage group button is clicked. Try having many groups and you click the last manage group. After clicking and redirecting to the page, you are now at the bottom at the page.`

can you please elaborate a bit more on this @ekowidianto ?? I tried to reproduce the mentioned things here and seems like it's not necessarily going down to the bottom

Reply:
done